### PR TITLE
Restore unit metadata; update controlled vocabulary

### PR DESCRIPTION
**Pull request by jefferya**
Purpose:
* restore unit metadata facet (i.e., rolls back changes in #17 and #91) -- see below for UI change result
* update controlled vocabulary in ` config/controlled_vocabulary.yml ` with updated list of units

Address step number 1 of #537 by adding the unit metadata back to the facets.

### CODE REVIEW

File: config/controlled_vocabulary.yml [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+  - Grant Assist Program (Social Sciences / Humanities)
+  - Kule Folklore Centre
+  - Music, Department of
+  - Intonations
+  - Physics, Department of
+  - School of Library and Information Studies
+  - Secondary Education, Department of
+  - University of Alberta Archives
+  - University of Alberta Library
+  - University of Alberta Library Open Resources
```
Comment (seanluyk) [inline config/controlled_vocabulary.yml:20]: This one can be removed, my bad

Comment (jefferya) [reply config/controlled_vocabulary.yml]: @seanluyk thanks, I've removed `Open Resources`

File: config/controlled_vocabulary.yml [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+  - Grant Assist Program (Social Sciences / Humanities)
+  - Kule Folklore Centre
+  - Music, Department of
+  - Intonations
+  - Physics, Department of
+  - School of Library and Information Studies
+  - Secondary Education, Department of
+  - University of Alberta Archives
+  - University of Alberta Library
+  - Writing Stick
```
Comment (jefferya) [inline config/controlled_vocabulary.yml:20]: @seanluyk Double checking, is this list of units in Avalon correct? Checking if any changes before asking the dev team for review.

Comment (seanluyk) [reply config/controlled_vocabulary.yml:20]: @jefferya this looks good to me with a recent addition needed for "Biological Sciences, Department of", but I'm flagging @anayram to make sure we're not missing anything.

**Comment by jefferya** [comment]
Unit added back to search facets:
Screenshot from 2020-08-06 07-20-13

**Pull request merged**